test/fw/ptl/lib/ptl_service.py

```diff
@@ -1458,3 +1458,42 @@ def cleanup_files(self):
         for dyn_files in self.dyn_created_files:
             self.du.rm(path=dyn_files, sudo=True, force=True)
         self.dyn_created_files = []
+
+    def isUp(self, inst, max_attempts=None):
+        """
+        Check for daemons up
+        """
+        if max_attempts is None:
+            max_attempts = self.ptl_conf['max_attempts']
+        for _ in range(max_attempts):
+            rv = self._isUp(inst)
+            if rv:
+                break
+            time.sleep(1)
+        return rv
+
+    def signal(self, inst, sig):
+        """
+        Send signal to daemons
+        """
+        if inst.__class__.__name__ == "Scheduler":
```

Comment:
I think you can replace all this ifs with single line:
`self.logger.info(f"{inst.__class__.__name__} sent signal {sig}")`.

Reply:
So f-strings are amazing.  The problem is they were added in 3.6, and one of the Ubuntu distros only supports 3.5 (I think it is 16).  Until we stop supporting it, we can't use them :(

Reply:
Okay.. then we can use .format().